frontend/src/routes/Routes.tsx

```diff
@@ -116,7 +116,7 @@ const ViewFacility = React.lazy(() => import(/* webpackChunckName: "ViewFacility
 const ViewMyBookings = React.lazy(() => import(/* webpackChunckName: "ViewMyBooking" */ './FacilityBooking/MyBookings'))
 const ViewBooking = React.lazy(() => import(/* webpackChunckName: "ViewBooking" */ './FacilityBooking/ViewBooking'))
 const CreateBooking = React.lazy(
-  () => import(/* webpackChunckName: "CreateBooking" */ './FacilityBooking/CreateBooking'),
+  () => import(/* webpackChunckName: "CreateBooking" */ './FacilityBooking/CreateBooking/index'),
```

Comment:
hmm /index? is this the way to add a param? not too sure. can verify.

Reply:
i think this is the file path not the route, though /index go access index file isn't necessary